src/code/audio/code_8019AF00.c

```diff
@@ -217,768 +236,893 @@ s8 gUnderwaterSfxReverbAdd = 0;
+    SEQ_FLAG_FANFARE,                     // NA_BGM_BALLAD_OF_THE_WIND_FISH
+    SEQ_FLAG_RESTORE,                     // NA_BGM_SONG_OF_SOARING
+    0,                                    // NA_BGM_MILK_BAR_DUPLICATE
+    0,                                    // NA_BGM_FINAL_HOURS
+    SEQ_FLAG_FANFARE,                     // NA_BGM_MIKAU_RIFF
+    SEQ_FLAG_FANFARE,                     // NA_BGM_MIKAU_FINALE
+    0,                                    // NA_BGM_FROG_SONG
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_SONATA
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_LULLABY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_NEW_WAVE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_ELEGY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_OATH
+    0,                                    // NA_BGM_MAJORAS_LAIR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_LULLABY_INTRO
+    SEQ_FLAG_FANFARE,                     // NA_BGM_OCA_GUITAR_BASS_SESSION
+    SEQ_FLAG_FANFARE,                     // NA_BGM_PIANO_SESSION
+    SEQ_FLAG_FANFARE,                     // NA_BGM_INDIGO_GO_SESSION
+    SEQ_FLAG_ENEMY,                       // NA_BGM_SNOWHEAD_TEMPLE
+    SEQ_FLAG_ENEMY,                       // NA_BGM_GREAT_BAY_TEMPLE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_NEW_WAVE_SAXOPHONE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_NEW_WAVE_VOCAL
+    0,                                    // NA_BGM_MAJORAS_WRATH
+    0,                                    // NA_BGM_MAJORAS_INCARNATION
+    0,                                    // NA_BGM_MAJORAS_MASK
+    SEQ_FLAG_FANFARE,                     // NA_BGM_BASS_PLAY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_DRUMS_PLAY
+    SEQ_FLAG_FANFARE,                     // NA_BGM_PIANO_PLAY
+    SEQ_FLAG_ENEMY,                       // NA_BGM_IKANA_CASTLE
+    0,                                    // NA_BGM_GATHERING_GIANTS
+    SEQ_FLAG_FANFARE_KAMARO,              // NA_BGM_KAMARO_DANCE
+    0,                                    // NA_BGM_CREMIA_CARRIAGE
+    SEQ_FLAG_FANFARE,                     // NA_BGM_KEATON_QUIZ
+    0,                                    // NA_BGM_END_CREDITS_1
+    0,                                    // NA_BGM_OPENING_LOOP
+    0,                                    // NA_BGM_TITLE_THEME
+    SEQ_FLAG_FANFARE,                     // NA_BGM_DUNGEON_APPEAR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_WOODFALL_CLEAR
+    SEQ_FLAG_FANFARE,                     // NA_BGM_SNOWHEAD_CLEAR
+    0,                                    //
+    0,                                    // NA_BGM_INTO_THE_MOON
+    0,                                    // NA_BGM_GOODBYE_GIANT
+    0,                                    // NA_BGM_TATL_AND_TAEL
+    0,                                    // NA_BGM_MOONS_DESTRUCTION
+    0,                                    // NA_BGM_END_CREDITS_2
```

Comment:
I suppose this is a candidate for BGM tables?

Reply:
Yes, I've 100% been brainstorming this. The main issue is that there's already a sequence table `gSequenceTable` which contains info like seq sizes, cache policy:

Here's what MM's looks like
https://github.com/engineer124/mm/blob/a2319c321f167377904c54c749fd79c763f445fd/audio_tables.c#L90

And here's what OoT's table looks like with the proper enums/macros:
https://github.com/MNGoldenEagle/oot/blob/de0d45dd7b61239212715f3ae9f768e0d4464d42/include/tables/sequence_table.h

Ideally, we could also include the seqFlags and enum definitions in there. The issue is that the currently proposed OoT extraction would generate these tables as part of audio extraction, and not be something committed to the repo, which is great for things like not having to manually adjust/calculate the size of the file. But then we can't add the enum definition/seqFlags to the table. So do we have two separate tables?